### PR TITLE
SAK-49000 - Remove duplicate id in Site Manage Overview

### PR DESCRIPTION
**Pull request by csev**

### CODE REVIEW

**Review by adrianfish — CHANGES_REQUESTED** [review]
I'd prefer if we changed these to use lower case with hypens. move -> move-1. This mixture of capitalisation and underscores just doesn't look very coherent to me. We could, perhaps, make them more useful by calling them "lefttool-move-header" and righttool-move-header, something like that, so we have more idea straight away what they are.

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm [hunk 1]
```diff
@@ -33,17 +33,17 @@
                 <table class="manageTable table table-bordered table-hover" cellspacing="0">
                     <thead>
                         <tr>
-                            <th id="move" class="text-center">$tlang.getString("manover.move")</th>
-                            <th id="Title">$tlang.getString("manover.title")</th>
-                            <th id="Remove" class="text-center">$tlang.getString("manover.remove")</th>
+                            <th id="Move_1" class="text-center">$tlang.getString("manover.move")</th>
+                            <th id="Title_1">$tlang.getString("manover.title")</th>
+                            <th id="Remove_1" class="text-center">$tlang.getString("manover.remove")</th>
```
Comment (adrianfish) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm:38]:
```suggestion
                            <th id="leftcolumn-move-header" class="text-center">$tlang.getString("manover.move")</th>
                            <th id="leftcolumn-title-header">$tlang.getString("manover.title")</th>
                            <th id="leftcolumn-remove-header" class="text-center">$tlang.getString("manover.remove")</th>
```

**Comment by csev** [comment]
@adrianfish - There are three spots that need fixing - I will fix them.

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm [hunk 2]
```diff
@@ -33,17 +33,17 @@
                 <table class="manageTable table table-bordered table-hover" cellspacing="0">
                     <thead>
                         <tr>
-                            <th id="Move_1" class="text-center">$tlang.getString("manover.move")</th>
-                            <th id="Title_1">$tlang.getString("manover.title")</th>
-                            <th id="Remove_1" class="text-center">$tlang.getString("manover.remove")</th>
+                            <th id="leftcolumn-move-header" class="text-center">$tlang.getString("manover.move")</th>
+                            <th id="leftcolumn-title-header">$tlang.getString("manover.title")</th>
+                            <th id="leftcolumn-remove-header" class="text-center">$tlang.getString("manover.remove")</th>
```
Comment (adrianfish) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm:38]:
```suggestion
                            <th id="left-column-move-header" class="text-center">$tlang.getString("manover.move")</th>
                            <th id="left-column-title-header">$tlang.getString("manover.title")</th>
                            <th id="left-column-remove-header" class="text-center">$tlang.getString("manover.remove")</th>
```

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm [hunk 3]
```diff
@@ -33,17 +33,17 @@
...
                         </tr>
                     </thead>
                     <tbody>
                         #foreach ($tool in $tools)
                         #set ($toolCount=$toolCount + 1)
                         #set ($returnval = $toolIds.add($tool.Tool.Id)) <!-- This needs to be set b/c .add returns boolean-->
                             <tr>
-                                <td headers="Move_1" class="buttonCol">
+                                <td headers="leftcolumn-move-header" class="buttonCol">
```
Comment (adrianfish) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm:46]:
```suggestion
                                <td headers="left-column-move-header" class="buttonCol">
```

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm [hunk 4]
```diff
@@ -55,10 +55,10 @@
                                         </a>
                                     #end
                                 </td>
-                                <td headers="Title_1">
+                                <td headers="leftcolumn-title-header">
```
Comment (adrianfish) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm:58]:
```suggestion
                                <td headers="left-column-title-header">
```

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm [hunk 5]
```diff
@@ -55,10 +55,10 @@
...
                                     <span class="titleSpan">$formattedText.escapeHtml($tool.Title)</span>
                                 </td>
-                                <td headers="Remove_1" class="removeCol">
+                                <td headers="leftcolumn-remove-header" class="removeCol">
```
Comment (adrianfish) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm:61]:
```suggestion
                                <td headers="left-column-remove-header" class="removeCol">
```

**Review by adrianfish — APPROVED** [review]
I've just added a couple of changes. leftcolumn -> left-column. Thanks for making the changes.

**Comment by csev** [comment]
Thanks

**Pull request merged**